src/main/java/com/bawnorton/impirium/ImPIriumClient.java

```diff
@@ -0,0 +1,60 @@
+package com.bawnorton.impirium;
+
+import com.mojang.brigadier.arguments.DoubleArgumentType;
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.client.command.v2.ClientCommandManager;
+import net.fabricmc.fabric.api.client.command.v2.ClientCommandRegistrationCallback;
+import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+public class ImPIriumClient implements ClientModInitializer {
+	private boolean showedWarning;
+
+	@Override
+	public void onInitializeClient() {
+		ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
+			showedWarning = false;
+		});
+
+		ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> {
+			MinecraftClient client = MinecraftClient.getInstance();
+			boolean singleplayer = client.isInSingleplayer();
```

Comment:
Just return, no need to register the same command twice

Reply:
sorry for late response, fixed